jquery.fitvids.js

```diff
@@ -9,7 +9,20 @@
 *
 */
 
-;(function( $ ){
+!function(factory) {
```

Comment:
please keep the style of the IIFE. The semicolon actually makes sense: https://github.com/davatron5000/FitVids.js/pull/192.